### ISSUE TITLE
I have issues in training ( evaluation problem! )

### ISSUE THREAD

**Issue by richard0326** [andy-yun/pytorch-0.4-yolov3#71]
env : win 10, pytorch 0.4.1, cuda 9.0

cfg changed : copied yolo_v3.cfg, batch=8 subdivision=4
                       last conv's filters=18, yolo layer classes=1

when the training reaches 10. It evaluates and get error...
I have tried '-n, -l, -i' commands but it failes... -n passes the evalutation but failes on detect.py
![1](https://user-images.githubusercontent.com/18134703/58849991-876a1100-86c7-11e9-914f-0ae42e71106f.PNG)

**Comment by richard0326** [comment]
what should I do for evaluation??

**Comment by andy-yun** [comment]
@richard0326  You should modify the filters in every [convolutional] before [yolo]:

[convolutional]
filters=18       # (classes+5)*num_anchors

However, in my opinion, that is not good condition if classes=1 because BCELoss  does not affect the training loss.

**Comment by richard0326** [comment]
Thank you! 
I have another problem but this one is solved! 

I don't know the BCELoss but its works for now!
Thank you!
